srcpkgs/mesa-demos/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'mesa-demos'
 pkgname=mesa-demos
-version=8.4.0
-revision=3
-build_style=gnu-configure
-configure_args="--with-system-data-files"
+version=9.0.0
+revision=1
+build_style=meson
+configure_args="-Dwith-system-data-files=true"
 hostmakedepends="pkg-config"
 makedepends="libXext-devel MesaLib-devel glu-devel glew-devel freetype-devel libfreeglut-devel"
 short_desc="Mesa 3D demos and tools"
 maintainer="John <me@johnnynator.dev>"
 license="MIT"
```

Comment:
The tarball doesn't seem to include any sort of license file, so I'm not sure what to do about this.

Reply:
maybe it can be grabbed from a comment in a file?

Reply:
The thing is, there seem to be various copyright notices scattered across different files...

Reply:
ask in irc